2023-01-13 20:21:32,790 INFO    MainThread:3010 [wandb_setup.py:_flush():68] Configure stats pid to 3010
2023-01-13 20:21:32,791 INFO    MainThread:3010 [wandb_setup.py:_flush():68] Loading settings from /home/dahala/.config/wandb/settings
2023-01-13 20:21:32,791 INFO    MainThread:3010 [wandb_setup.py:_flush():68] Loading settings from /home/dahala/extremevaluelandslides/wandb/settings
2023-01-13 20:21:32,791 INFO    MainThread:3010 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2023-01-13 20:21:32,791 INFO    MainThread:3010 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-01-13 20:21:32,791 INFO    MainThread:3010 [wandb_init.py:_log_setup():478] Logging user logs to /home/dahala/extremevaluelandslides/wandb/run-20230113_202132-yz42aj6y/logs/debug.log
2023-01-13 20:21:32,791 INFO    MainThread:3010 [wandb_init.py:_log_setup():479] Logging internal logs to /home/dahala/extremevaluelandslides/wandb/run-20230113_202132-yz42aj6y/logs/debug-internal.log
2023-01-13 20:21:32,791 INFO    MainThread:3010 [wandb_init.py:_jupyter_setup():428] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x2b7bf93d33d0>
2023-01-13 20:21:32,791 INFO    MainThread:3010 [wandb_init.py:init():518] calling init triggers
2023-01-13 20:21:32,792 INFO    MainThread:3010 [wandb_init.py:init():521] wandb.init called with sweep_config: {}
config: {}
2023-01-13 20:21:32,792 INFO    MainThread:3010 [wandb_init.py:init():537] re-initializing run, found existing run on stack: adybffui
2023-01-13 20:21:32,794 INFO    MainThread:3010 [wandb_run.py:_finish():1844] finishing run dahal/Extreme Value/adybffui
2023-01-13 20:21:32,794 INFO    MainThread:3010 [wandb_run.py:_atexit_cleanup():2070] got exitcode: 0
2023-01-13 20:21:32,795 INFO    MainThread:3010 [wandb_run.py:_restore():2053] restore
2023-01-13 20:21:32,795 INFO    MainThread:3010 [wandb_run.py:_restore():2059] restore done
2023-01-13 20:22:13,562 INFO    MainThread:3010 [wandb_init.py:init():571] starting backend
2023-01-13 20:22:13,562 INFO    MainThread:3010 [wandb_init.py:init():575] setting up manager
2023-01-13 20:22:13,565 INFO    MainThread:3010 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2023-01-13 20:22:13,591 INFO    MainThread:3010 [wandb_init.py:init():582] backend started and connected
2023-01-13 20:22:13,620 INFO    MainThread:3010 [wandb_run.py:_label_probe_notebook():1202] probe notebook
2023-01-13 20:22:13,621 INFO    MainThread:3010 [wandb_run.py:_label_probe_notebook():1212] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-01-13 20:22:13,621 INFO    MainThread:3010 [wandb_init.py:init():670] updated telemetry
2023-01-13 20:22:13,656 INFO    MainThread:3010 [wandb_init.py:init():710] communicating run to backend with 60.0 second timeout
